packages/ts/generator-typescript-plugin-backbone/test/complex-hierarchy/fixtures/ComplexHierarchyEndpoint.snap.ts

```diff
@@ -1,4 +1,4 @@
-import type ComplexHierarchyModel_1 from "./com/vaadin/fusion/parser/plugins/backbone/complexhierarchy/models/ComplexHierarchyModel.js";
-import client_1 from "./connect-client.default.js";
+import type ComplexHierarchyModel_1 from "./com/vaadin/fusion/parser/plugins/backbone/complexhierarchy/models/ComplexHierarchyModel";
```

Comment:
not very important, but why do we have these `_1` suffixes?

Reply:
It is the result of TypeScript's `createUniqueName` function. It creates an identifier that will be unique across the source file. I use it everywhere to avoid naming collision.